src/pad.c

```
@@ -709,34 +727,31 @@ static void padRumbleArm(int durationMs, unsigned char level)
+    padRumbleArm(RUMBLE_TAP_MS, RUMBLE_TAP_LEVEL, 0);
 }
 
-/** Light tick for a cursor move in the GAME LIST. Same event as padRumbleTap() -- both are
- *  sfxPlay(SFX_CURSOR) -- but firmer and slightly longer, because hardware says the two screens do not
- *  feel alike at identical settings. sound.c picks between them off the current GUI screen: sfxPlay()
- *  is handed only an sfx id, so the screen cannot be inferred any deeper than the hook. */
+/** Game-list variant of padRumbleTap(). The two are IDENTICAL since the RETRO retune -- the old
+ *  firmer/longer list split existed to compensate for a 60ms pulse dying inside the list's slow
+ *  art-loading frames, which a 240ms thump no longer suffers. The seam is kept (sound.c still picks
+ *  per screen off guiGetCurrentScreen) so re-splitting after HW feedback is a one-constant change. */
 void padRumbleTapList(void)
```

Cppcheck (2.21.0)

[style] The function 'padRumbleTapList' is never used.

(unusedFunction)